blog/_posts/2021-11-12-ahed-project.md

```diff
@@ -0,0 +1,392 @@
+
+As a recent addition to the ADS Team supporting curation efforts and assisting in collection management, this appealed to my interests as I have recently begun honing my Python skills, and learning new tools such as Jupyter Notebook and OpenRefine. Jupyter Notebook is especially useful for new Python users because it helps break up scripts into more manageable blocks (cells) and I can include my notes, findings, and documentation along the way.
+ 
+In this documentation I intend to outline the goals I established, the steps I took to accomplish them, and lessons learned. To accomplish this project, I used a combination of my own knowledge and expertise, I read API Documentation, searched online for solutions as needed, and I also collaborated with team members to debug and learn the ins and outs of the ADS API.
+
+### Project Outline and Goals
+
+The source data used in this project was an excel spreadsheet provided of AHED’s bibliographic holdings (with metadata for Authors, Org code, Title, Journal information, and DOI if available), which was split up into three sheets by Branch (Astrophysics Branch, the Planetary Systems Branch, and the Exobiology Branch). 
+
+<b>Sample image of original excel file, showing items in three tabs for the AHED Branches</b>
+
+<img width="1004" alt="Screen Shot 2021-11-04 at 11 44 23 AM" src="https://user-images.githubusercontent.com/31739067/140366721-e2721366-94d7-4651-a06d-17e7a10483e4.png"> 
+
+The main overall goal was to match each publication with an ADS bibcode if it exists. In order to accomplish this, my team recommended I use the ADS API to match based on DOIs (for those provided), then reference strings, and then fill in the rest by Title. I split up my overall goal into four major tasks, and for each of these phases I created a new Jupyter Notebook and outlined the steps.
+1. [(Notebook 1): Match AHED to ADS Items by DOI](#match-by-doi)
+2. [(Notebook 2): Match AHED to ADS Items by Reference Strings](#match-by-ref)
+3. [(Notebook 3): Match AHED to ADS Items by Title](#match-by-title)
+4. [(Notebook 4): Curate missing items and create ADS Libraries](#ads-libs)
+
+## <a name="match-by-doi">Task 1 (Notebook 1): Match AHED to ADS Items by DOI</a>
+
+Create a version of the AHED spreadsheet that has a column "bibcode" added to the right of the DOI column. For those publications we are able to match to ADS records, this column will list these bibcodes, otherwise "NA".
```

Comment:
"I first created a version of the AHED spreadsheet that has..."

Reply:
And I'd actually add another sentence at the beginning of this paragraph - for all of your other big sections, you start with something like "My goal is..." and I think you'd benefit from something like that here. What about "My first goal is to match items in the AHED spreadsheet to ADS records by DOI."